lib/osm_community_index/local_chapter.rb

```diff
@@ -0,0 +1,64 @@
+module OsmCommunityIndex
+  class LocalChapter
+    attr_reader :id, :url
+
+    def initialize(id, url)
+      @id = id
+      @url = url
+    end
+
+    def self.local_chapters
+      @chapters = init_local_chapters
+    end
+
+    def self.init_local_chapters
+      raw_local_chapters = load_raw_local_chapters
+      local_chapters = []
+      raw_local_chapters.each do |chapter|
```

Comment:
`map` rather than `foo = [] ... each ... foo.push(bar)` to make this easier to read